# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmCodegenTypes.h
# Commit: ce799cb566ac
# Full Hash: ce799cb566ac0fb95468118608132be11db6a359
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-24 21:54:20
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r c10609d434ae -r ce799cb566ac js/src/wasm/WasmCodegenTypes.h
--- a/js/src/wasm/WasmCodegenTypes.h	Thu Mar 24 17:40:32 2022 +0000
+++ b/js/src/wasm/WasmCodegenTypes.h	Thu Mar 24 17:56:50 2022 +0000
@@ -604,7 +604,7 @@
   uint32_t tableFunctionBaseGlobalDataOffset() const {
     MOZ_ASSERT(isTable());
     return u.table.globalDataOffset_ +
-           offsetof(TableInstanceData, functionBase);
+           offsetof(TableInstanceData, elements);
   }
   TypeIdDesc wasmTableSigId() const {
     MOZ_ASSERT(which_ == WasmTable);